<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>-60</y>
      <width>720</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="Live_horizontalSlider">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>440</y>
      <width>800</width>
      <height>5</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="play_pushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>430</y>
      <width>30</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>▶</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pause_pushButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>430</y>
      <width>30</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>■</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reverse_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>30</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>◀</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_pushButton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>430</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QWidget" name="openGLContainer" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>961</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
